src/components/SizeThumbnails.tsx

```diff
@@ -1,10 +1,10 @@
 import { ReactElement, useState, useEffect } from "react";
-import { theme } from "@/theme";
-
+import { BaseIconButton, theme } from "@gliff-ai/style";
 import { Card } from "@material-ui/core";
 
 import { makeStyles } from "@material-ui/core/styles";
-import TooltipButton from "@/components/TooltipButton";
+
+import { thumbnailSizes, ThumbnailSizes } from "@/components/Tooltips";
```

Comment:
Why is the thumbnail size information being moved into a Tooltips file?

Reply:
Does a 'thumbnails' file make more sense?

Reply:
The idea was to have all the tooltips in one file because the thumbnail size information is being used as tooltips  similar to other tooltips information that the Tooltips file currently holds.

Happy to create a new file just for the thumbnails size information though